.circleci/config.yml

```diff
@@ -63,23 +63,29 @@ references:
    run:
      name: Make Documentation
      command: |
-       # sudo apt-get install pandoc
+       # First run the same pipeline as Read-The-Docs
        sudo apt-get update && sudo apt-get install -y cmake
-       pip install -r requirements.txt --user
        sudo pip install -r docs/requirements.txt
-       pip install -r requirements-extra.txt --user  # for doctesting loggers etc.
-       # sphinx-apidoc -o ./docs/source ./pytorch_lightning **/test_* --force --follow-links
        cd docs; make clean; make html --debug --jobs 2 SPHINXOPTS="-W"
-       make doctest; make coverage
+
+  test_docs: &test_docs
+   run:
+     name: Testing Documentation
+     command: |
+       cd docs; make doctest; make coverage
 
 jobs:
 
   Build-Docs:
     docker:
       - image: circleci/python:3.7
+    environment:
+        - TORCH_VERSION: "torch"
     steps:
       - checkout
       - *make_docs
+      - *install_deps
+      - *test_docs
```

Comment:
it doesn't seem to install deps first, maybe they don't run in sequence. did you try removing the * ?

Reply:
I forget about our setting the mock setting in config.py which doe not use mock for CorcleCI